Sage/carbon
# fix(search): address alignment styling bug with search button - FE-3193

### Proposed behaviour
Fix the styling bug with the search button. A change elsewhere in the code base has meant that the button now hangs 3px below the search textbox.

<img width="491" alt="Screenshot 2020-10-06 at 15 36 50" src="https://user-images.githubusercontent.com/56251247/95216329-d6f3ac00-07e9-11eb-9b51-cb3a655dfaed.png">

### Current behaviour
The search button hangs 3px below the search texbox.

<img width="464" alt="Screenshot 2020-10-06 at 15 37 09" src="https://user-images.githubusercontent.com/56251247/95216379-e541c800-07e9-11eb-910d-1247bdc5f992.png">

### Checklist



### Additional context
As an additional task I have removed the redundant documentation and added a semi-colon to the `Quick Start` section in the MDX example.

### Testing instructions
- `git checkout FE-3193-fix_search_button_height`
- `npm start` to start Storybook
- Navigate to Design System -> Search
- Visit an example with a search button 
- Search button should be flush with the bottom of the search textbox
